src/Converter.mjs

```diff
@@ -49,10 +53,72 @@ class Converter {
 
     console.debug(`Converting to 3D Tiles...`)
     let boundingBox = BoundingBox.fromBoundingBoxes(boundingBoxes)
-    let tileset = await this.convertCityObjects(cityObjects, boundingBox)
+
+    let tileBoxes = this._getTilingScheme(boundingBox)
+    let tiles = await Promise.all(tileBoxes.map(box => {
+      const objects = []
+
+      console.log(`  Converting tile ${box.index[0]}/${box.index[1]} ...`)
+
+      for (const obj of cityObjects) {
+        if (box.bbox.contains(obj.getCenter())) {
+          objects.push(obj)
+        }
+      }
+
+      return this.convertCityObjects(objects, box.bbox)
+    }))
 
     console.debug(`Writing 3D Tiles...`)
-    await tileset.writeToFolder(outputFolder)
+
+    let data = {
+      asset: {
+        version: '0.0'
+      },
+      geometricError: 9999,
+      root: {
+        refine: 'ADD',
+        boundingVolume: {
+          region: [
+            boundingBox.min.longitude,
+            boundingBox.min.latitude,
+            boundingBox.max.longitude,
+            boundingBox.max.latitude,
+            0,
+            boundingBox.max.height,
+          ]
+        },
+        geometricError: 999,
+        children: []
+      }
+    }
+
+    await Promise.all(tiles.map(async (tileset, i) => {
+      let [ilat, ilon] = tileBoxes[i].index
+      let path = Path.join(ilat.toString(), ilon.toString());
+
+      console.log(`  Writing tile ${ilat}/${ilon} ...`)
+      data.root.children.push({
+        refine: 'ADD',
+        boundingVolume: {
+          region: [
+            tileBoxes[i].bbox.min.longitude,
+            tileBoxes[i].bbox.min.latitude,
+            tileBoxes[i].bbox.max.longitude,
+            tileBoxes[i].bbox.max.latitude,
+            tileBoxes[i].bbox.min.height,
+            tileBoxes[i].bbox.max.height,
+          ]
+        },
+        geometricError: 99,
+        content: {
+          uri: Path.join(".", path, "tileset.json").toString().replaceAll('\\', '/'),
+        }
+      })
+      await tileset.writeToFolder(Path.join(outputFolder, path))
+    }))
+
+    fsExtra.outputFile(Path.join(outputFolder, "tileset.json"), JSON.stringify(data, null, 2))
```

Comment:
Previously this code was in `Tileset.writeToFolder()`, wondering what we should do about it..

I'll think about it a bit.